src/env.h

```diff
@@ -20,34 +20,34 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 #ifndef SRC_ENV_H_
-#define SRC_ENV_H_
-
-#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
-
-#include "aliased_buffer.h"
-#if HAVE_INSPECTOR
-#include "inspector_agent.h"
-#include "inspector_profiler.h"
-#endif
-#include "handle_wrap.h"
-#include "node.h"
-#include "node_binding.h"
-#include "node_http2_state.h"
-#include "node_main_instance.h"
-#include "node_options.h"
-#include "req_wrap.h"
-#include "util.h"
-#include "uv.h"
-#include "v8.h"
-
-#include <array>
-#include <atomic>
-#include <cstdint>
-#include <functional>
-#include <list>
-#include <unordered_map>
-#include <unordered_set>
-#include <vector>
+# define SRC_ENV_H_
+
+# if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
```

Comment:
Hm… I guess we might want to introduce exceptions for header guards, i.e. the `SRC_..._H_` ones and `NODE_WANT_INTERNALS` – otherwise we get a huge diff here, and tbh having to indent all “meaningful” preprocessor directives in a file looks a bit odd to me?

Reply:
For things like `#define SRC_ENV_H_` the linter could check for a define with an identifier and without a replacement. For `#if defined(NODE_WANT_INTERNALS)...` I wouldn't be too sure on how to go about that. Is `#if defined...` used just for header guards?